docs/customization/api.rst

```diff
@@ -532,6 +532,54 @@ And here we go, now your response should look like this:
         //...
     }
 
+Configuring endpoints using yaml
+--------------------------------
+
+To remove an endpoint from the API using YAML you need to specify the operation of which resource
+should be removed in ``config/api_platform/config.yaml``.
+
+If you want to remove, for example, the admin ``GET`` endpoint of ``Zones``, you need to use the ``(unset)`` key in its yaml config.
+
+.. code-block:: yaml
+
+    '%sylius.model.zone.class%':
+        collectionOperations:
+            admin_get (unset): ~
+
+
+Using the ``(unset)`` key you can also remove filters you don't need.
+
+.. code-block:: yaml
+
+    '%sylius.model.product.class%':
+        collectionOperations:
+            shop_get:
+                filters (unset):
+                    - app.product_new_filter
+
+If you need to add a new filter in the yaml configuration, simply add this kind of code to ``config/api_platform/config.yaml``.
+
+.. code-block:: yaml
+
+    '%sylius.model.product.class%':
+        collectionOperations:
+            shop_get:
+                filters:
+                    - app.product_new_filter
+
+
+To add a new operation, just specify it in the config file.
+
+.. code-block:: yaml
+
+    '%sylius.model.channel.class%':
+        collectionOperations
+            my_new_operation:
+                method: GET
+                path: /shop/channels
+                normalization_context:
+                    groups: ['shop:channel:read']
+
```

Comment:
IIRC you can also add a case with overwriting the entire endpoint configuration (you firstly should unset operation and after that add new one).

Reply:
good point, added a way to overwrite existing endpoint (we dont need to unset it)